rtp.h

```diff
@@ -195,4 +195,38 @@ int janus_rtp_skew_compensate_audio(janus_rtp_header *header, janus_rtp_switchin
  * @returns 0 if no compensation is needed, -N if a N packets drop must be performed, N if a N sequence numbers jump has been performed */
 int janus_rtp_skew_compensate_video(janus_rtp_header *header, janus_rtp_switching_context *context, gint64 now);
 
+/* Payload types we'll offer internally */
+#define OPUS_PT		111
+#define ISAC32_PT	104
+#define ISAC16_PT	103
+#define PCMU_PT		0
+#define PCMA_PT		8
+#define G722_PT		9
+#define VP8_PT		96
+#define VP9_PT		101
+#define H264_PT		107
```

Comment:
These payload types here should be statically defined in `rtp.c` and not here, I think, especially considering they're not used anywhere else in the code (nor should they I think). If you see any reason for having them defined in the header, instead, then they should be prefixed (e.g., `JANUS_OPUS_PT` or `JANUS_AUDIO_OPUS_PT`).